application/F3DOptionsTools.cxx

```diff
@@ -0,0 +1,565 @@
+      {"cells", "c", "Use an array from the cells", "<bool>", "1"},
+      {"range", "", "Custom range for the coloring by array", "<min,max>", ""},
+      {"bar", "b", "Show scalar bar", "<bool>", "1" },
+      {"colormap-file", "", "Specify a colormap image", "<filePath/filename/fileStem>", ""},
+      {"colormap", "", "Specify a custom colormap (ignored if \"colormap-file\" is specified)", "<color_list>", ""},
+      {"volume", "v", "Show volume if the file is compatible", "<bool>", "1"},
+      {"inverse", "i", "Inverse opacity function for volume rendering", "<bool>", "1"} } },
+  {"Camera",
+    { {"camera-position", "", "Camera position (overrides camera direction and camera zoom factor if any)", "<X,Y,Z>", ""},
+      {"camera-focal-point", "", "Camera focal point", "<X,Y,Z>", ""},
+      {"camera-view-up", "", "Camera view up", "<X,Y,Z>", ""},
+      {"camera-view-angle", "", "Camera view angle (non-zero, in degrees)", "<angle>", ""},
+      {"camera-direction", "", "Camera direction", "<X,Y,Z>", ""},
+      {"camera-zoom-factor", "", "Camera zoom factor (non-zero)", "<factor>", ""},
+      {"camera-azimuth-angle", "", "Camera azimuth angle (in degrees), performed after other camera options", "<angle>", ""},
+      {"camera-elevation-angle", "", "Camera elevation angle (in degrees), performed after other camera options", "<angle>", ""},
+      {"camera-orthographic", "", "Use an orthographic camera", "<bool>", "1"} } },
+
+#if F3D_MODULE_RAYTRACING
+  {"Raytracing",
+    { {"raytracing", "r", "Enable raytracing", "<bool>", "1"},
+      {"samples", "", "Number of samples per pixel", "<samples>", ""},
+      {"denoise", "d", "Denoise the image", "<bool>", "1"} } },
+#endif
+  {"PostFX (OpenGL)",
+    { {"translucency-support", "p", "Enable translucency support, implemented using depth peeling", "<bool>", "1"},
+      {"ambient-occlusion", "q", "Enable ambient occlusion providing approximate shadows for better depth perception, implemented using SSAO", "<bool>", "1"},
+      {"anti-aliasing", "a", "Enable anti-aliasing, implemented using FXAA", "<bool>", "1"},
+      {"tone-mapping", "t", "Enable Tone Mapping, providing balanced coloring", "<bool>", "1"},
+      {"final-shader", "", "Execute the final shader at the end of the rendering pipeline", "<GLSL code>", ""} } },
+  {"Testing",
+    { {"ref", "", "Reference", "<png file>", ""},
+      {"ref-threshold", "", "Testing threshold", "<threshold>", ""},
+      {"interaction-test-record", "", "Path to an interaction log file to record interactions events to", "<file_path>", ""},
+      {"interaction-test-play", "", "Path to an interaction log file to play interaction events from when loading a file", "<file_path>", ""} } }
+};
+
+/**
+ * True boolean options need to be filtered out in ParseCLIOptions
+ * This is the easiest, compile time way to do it
+ */
+static inline const std::set<std::string> CLIBooleans = {"version", "help", "readers-list", "scan-plugins"};
```

Comment:
Any reason to keep it ordered?  
A `std::array<std::string_view>` sounds more appropriate. The size of the array should be deduced by the compiler.

Reply:
> Any reason to keep it ordered?

Currently, we get options group in alphabetical order, I dont think we want that.

Reply:
no reason.